subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -109,119 +111,192 @@ static int update_job_execution(struct mqtt_client *const client,
 	return ret;
 }
 
-
-static int aws_fota_on_publish_evt(struct mqtt_client *const client,
-				   const u8_t *topic,
-				   u32_t topic_len,
-				   u32_t payload_len)
+/**
+ * @brief Parsing an AWS IoT Job Execution response recived on $next/get MQTT
+ *	  topic or notify-next if it is an valid response the program state is
+ *	  updated.
+ *
+ * @param[in] client Connected MQTT client instance
+ * @param[in] payload_len Length of the payload going to be read out from the
+ *	      MQTT message.
+ *
+ * @return 1 If successful otherwise an negative error code is returned.
```

Comment:
should be `a negative`.
there are a few other instances of this typo.

Reply:
Think I've addressed this issue now.